.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+# Summary
+
+## Expected Behavior
+
+## Current Behaviour
+
+## Steps to reproduce
+1.  
+2.
+3.
+
+## Screenshots (if appropriate)
+
+
+## Context (Environment)
```

Comment:
Do you think we should add details here as to what environment can be? For example, operating system, browser name and version, etc.

Reply:
added the new issue template with points regarding OS, browser, version and device (if the problem is with a specific smartphone).

Reply:
Thanks, Sahil, it looks good to me.